components/clarinet-sdk/vitest-helpers/src/formatCV.ts

```diff
@@ -0,0 +1,130 @@
+/*
+  This file could be implement in stacks.js
+  Format stacks.js Clarity Value into Clarity style readable string
+  eg:
+  `Cl.uint(1)` => u1
+  `Cl.list(Cl.uint(1))` => (list u1)
+*/
+
+import {
+  BufferCV,
+  ClarityType,
+  ClarityValue,
+  ListCV,
+  TupleCV,
+  principalToString,
+} from "@stacks/transactions";
+
+let byteToHex: string[] = [];
+for (let n = 0; n <= 0xff; ++n) {
+  const hexOctet = n.toString(16).padStart(2, "0");
+  byteToHex.push(hexOctet);
+}
+
+function formatSpace(space: number, depth: number, end = false) {
```

Comment:
We could use these from stacks.js if you want

Reply:
Yep, I will use it in v1 now that it's available. But v0.1 was already released like that so.. let's merge the PR as it is.
My process of releasing before it's merge isn't super conventional but well, it's a v0